Dockerfile

```
@@ -1,7 +1,7 @@
 # syntax=docker/dockerfile:1
 # Multistage build
 # Stage 0: Build
-FROM --platform=linux/amd64 node:16-alpine
+FROM --platform=linux/amd64 node:20-alpine
```

Hadolint (2.12.0)

[warning] Do not use --platform flag with FROM

(DL3029)